pkg/core/interop_neo_test.go

```diff
@@ -457,8 +457,43 @@ func TestAssetGetPrecision(t *testing.T) {
 	require.Equal(t, big.NewInt(int64(assetState.Precision)), precision)
 }
 
+func TestSecp256k1Recover(t *testing.T) {
+	v, context, chain := createVM(t)
+	defer chain.Close()
+
+	v.Estack().PushVal(vm.NewByteArrayItem([]byte{72, 101, 108, 108, 111, 87, 111, 114, 108, 100}))
+	v.Estack().PushVal(vm.NewBoolItem(false))
```

Comment:
I keep seeing these `vm.New*` wrappers, but I don't think they're needed here, `PushVal` should do the appropriate conversions internally.

Reply:
Fixed this at all places.